lib/jnpr/junos/decorators.py

```diff
@@ -119,14 +119,20 @@ def wrapper(*args, **kwargs):
                             if isinstance(ignore_warn, (str, unicode)):
                                 if not re.search(ignore_warn, err['message'],
```

Comment:
As I understood it, the expected behavior is that every warning message must match the value of `ignore_warning`. So, if the value of `ignore_warning` is a single string (such as `statement not found`, then every warning message in the response must match that value. If there are two warning messages with different values, and the user wants to ignore both of them, then they need to specify `ignore_warning` as a list of strings.

I believe my current code matches the behavior I've described above. If you see that it doesn't match this behavior or you disagree that this should be the expected behavior, then please do let me know.